crates/sui-types/src/crypto.rs

```diff
@@ -76,11 +76,7 @@ pub const DERIVATION_PATH_COIN_TYPE: u32 = 784;
 pub const DERVIATION_PATH_PURPOSE_ED25519: u32 = 44;
 pub const DERVIATION_PATH_PURPOSE_SECP256K1: u32 = 54;
 
-/// Default hash function for user-facing purposes, namely the ones that is also done by the wallet.
-pub type UserHash = Sha3_256;
-
-/// Default hash function for non-user-facing purposes, namely the ones not done by the wallet.
-pub type InternalHash = Blake2b256;
+pub type DefaultHash = Blake2b256;
```

Comment:
Add comment:
/// Blake2b_256 (output size = 32 bytes) is the default hash function for all Sui related operations including consensus and user-facing digests, such as publicKey to address and transactionID derivation.